spot-oa/oa/dns/dns_conf.json

```diff
@@ -27,23 +19,17 @@
         , "dns_qry_class" : 4
```

Comment:
it looks like these changes will affect feedback (the OA-to-ML interface) as well...

I need to incorporate those changes as well and we'll have to test with and without feedback after these changes